test/blocks/merch/mocks/fetch.js

```diff
@@ -6,7 +6,7 @@ import { priceLiteralsURL } from '../../../../libs/blocks/merch/merch.js';
 export async function mockFetch() {
   // this path allows to import this mock from tests for other blocks (e.g. commerce)
   const literals = JSON.parse(await readFile({ path: '../merch/mocks/literals.json' }));
-  const offers = JSON.parse(await readFile({ path: '../merch/mocks/offers.json' }));
+  const [offers, namedOffers] = JSON.parse(await readFile({ path: '../merch/mocks/offers.json' }));
```

Comment:
This looks a little bit complex to me, especially in json file (array of array and object). Maybe separate file for named offers?